fe/fe-core/src/test/java/com/starrocks/analysis/CreateFunctionStmtTest.java

```diff
@@ -0,0 +1,42 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.utframe.UtFrameUtils;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CreateFunctionStmtTest {
+    private ConnectContext ctx;
+
+    @Before
+    public void setUp() throws Exception {
+    }
+
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void testNormal() throws Exception {
+        ctx = UtFrameUtils.createDefaultCtx();
+        ctx.setDatabase("testDb");
+        ctx.setCluster("default_cluster");
+        compareAfterParse("CREATE FUNCTION f(INT, INT) RETURNS INT");
+        compareAfterParse("CREATE AGGREGATE FUNCTION f(INT, INT) RETURNS INT INTERMEDIATE INT");
+        compareAfterParse("CREATE FUNCTION f(INT, INT) RETURNS INT PROPERTIES (\"key\"=\"value\")");
```

Comment:
add a case for create table function

Reply:
Product Documentation should update(not include table funciton) https://docs.starrocks.com/zh-cn/main/sql-reference/sql-statements/data-definition/create-function#%E7%A4%BA%E4%BE%8B

Reply:
https://docs.starrocks.com/zh-cn/main/using_starrocks/JAVA_UDF#%E6%AD%A5%E9%AA%A4%E4%B8%89%E5%9C%A8-starrocks-%E4%B8%AD%E5%88%9B%E5%BB%BA-udtf-%E5%87%BD%E6%95%B0